textkit/transform/stem.py

```diff
@@ -0,0 +1,35 @@
+import click
+from nltk.stem import PorterStemmer
+from nltk.stem.lancaster import LancasterStemmer
+from nltk.stem.snowball import EnglishStemmer
+from nltk.stem.wordnet import WordNetLemmatizer
+from textkit.utils import read_tokens, output
+
+
+@click.command()
+@click.argument('tokens', type=click.File('r'), default=click.open_file('-'))
+@click.option('--algorithm', default='porter',
+              help='Choice of stemming algorithm: porter, lancaster, snowball, wordnet',
+              type=click.STRING,
+              show_default=True)
+def stem(tokens, algorithm):
+    '''Apply stemming to an iterable of tokens.'''
+    content = read_tokens(tokens)
+
+    stemmer_classes = {'porter': PorterStemmer,
```

Comment:
Does this require downloading some of the stemmer algorithms from NLTK? If so we need to add them to the download command.